MSBLOC.Core/Model/ProjectDetails.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace MSBLOC.Core.Model
+{
+    public class ProjectDetails
+    {
+        public string ProjectFile { get; }
+        public string CloneRoot { get; }
+        public string ProjectDirectory { get; }
+        public IReadOnlyDictionary<string, string> Paths => _paths;
```

Comment:
Shouldn't just be a cast

`public IReadOnlyDictionary<string, string> Paths => new ReadOnlyDictionary<string, string>(_paths);`